src/mscorlib/shared/README.md

```diff
@@ -0,0 +1,15 @@
+# .NET Core Runtime Shared Directory
+
+This directory contains the shared sources for System.Private.CoreLib. These are shared between dotnet/corert and dotnet/coreclr.
+
+The sources are synchronized with a mirroring tool that watches for new commits on either side and creates new pull requests (as @dotnet-bot) in the other repository.
+
+## Getting clean CI and merging the mirror PRs
+
+Once the mirror PR is created there is a high chance that the new code will require changes to get a clean CI. Any changes can be added to the PR by checking out the PR branch and adding new commits. Please follow the following guidelines for modifying these PRs.
```

Comment:
I would just say `there is a chance`. There is high chance today because of we are moving a bunch of stuff around. But once the dust settles, it should be much smoother - especially for bug-fix level changes.